apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -437,6 +438,9 @@
         }
 
         $(document).ready(function () {
+            $('.user-name').popup({
```

Comment:
Shall we use an `id` to identify the element rather than using a CSS class.
```suggestion
            $('.user-name').popup({
```

Reply:
Id is added instead of class name